src/components/Firebase/firebase.js

```diff
@@ -2,129 +2,260 @@ import firebase from 'firebase';
 import React from 'react';
 
 const config = {
-  apiKey: "AIzaSyAwyhl8Pi7drd3ls6bSaxWTYx3LnyyO7_Q",
-  authDomain: "devops-trivia-18a2c.firebaseapp.com",
-  databaseURL: "https://devops-trivia-18a2c.firebaseio.com",
-  projectId: "devops-trivia-18a2c",
-  storageBucket: "devops-trivia-18a2c.appspot.com",
-  messagingSenderId: "701353522538",
-  appId: "1:701353522538:web:e621e9c40ab4fbf2dee113",
-  measurementId: "G-SJ6TD2BJRW"
+  apiKey: process.env.REACT_APP_FIREBASE_API_KEY,
+  authDomain: "duu-trivia.firebaseapp.com",
+  databaseURL: "https://duu-trivia.firebaseio.com",
+  projectId: "duu-trivia",
+  storageBucket: "duu-trivia.appspot.com",
+  messagingSenderId: "503528390951",
+  appId: "1:503528390951:web:691b0e36f43d7209706902",
+  measurementId: "G-L05CNT2ZWP"
 };
 
-const ADMIN_REF_NAME = "quizzes";
-const NUM_QUESTIONS = 10;
+// const ADMIN_REF_NAME = "quizzes";
+// const NUM_QUESTIONS = 10;
 
 class Firebase {
   constructor() {
     firebase.initializeApp(config);
     this.auth = firebase.auth();
+    this.db = firebase.firestore();
   }
- 
+
   /**
    * @param {string} email
    * @param {string} password
    * @param {function} callback
    */
-  signInWithEmail = (email, password, callback) => {
+  signInWithEmail = (email, password, cb) => {
     this.auth.signInWithEmailAndPassword(email, password)
-      .then(() => callback(null))
-      .catch((error) => {
-        console.log(error);
+      .then((res) => {
+        cb({ success: true, msg: email });
+      })
+      .catch((err) => {
+        cb({ success: false, msg: err.message })
       });
   };
 
   /**
-   * @param {Object} quiz
+   * @param {String} name
    * @param {function} callback
    */
-  saveQuizFromAdmin = (quiz, callback) => {
-    let quizRef = firebase.database().ref().child(this.getCurrentFormattedDate()).child('admin');
-
-    // firebase doesn't allow insertion of arrays, so create children by auto id
-    var questionNum = 1;
-    quiz.forEach(QASet => {
-      quizRef.child(questionNum).set(QASet);
-      questionNum += 1;
-    });
+  createQuiz = (name, cb) => {
+    // create an empty round array
+    // 11 questions in case of bonus
+    const round = [];
+    for (let i = 0; i < 11; i++) {
+      round.push({
+        q: '',
+        a: '',
+        img: '',
```

Comment:
`question`, `answer`, `image`, let's use fully-qualified names for clean code reasons